TKeazirian.HTTPServer/Constants.cs

```diff
@@ -7,5 +7,6 @@ public static class Constants
         public const string Space = " ";
         public const string Status200 = "HTTP/1.1 200 OK";
         public const string Status404 = "HTTP/1.1 404 Not Found";
+        public static string Status301 = "HTTP/1.1 301 Moved Permanently";
```

Comment:
Looks like these status constants comprise three different elements. Do you think any or all of them are worth extracting to their own constants? Why or why not?

Reply:
I brought this up too; @tpetersen0308 do you think its in-scope for this PR? and if so, let's do something about it.

Reply:
I actually branched off of this branch and started working on it separately while waiting for review on this one.  So could it be separate?

Reply:
I don't see any reason it needs to be addressed in this PR 👍